HW02_B_Mimic_starter.Rmd

```diff
@@ -90,8 +116,15 @@ The above graph lets you see some colobrlind friendly palettes. For the graph be
 
 Now using the above mpg dataset, make this graph
 
-```{r graph 4 code}
-
+```{r graph 4 code, echo = T, include = TRUE}
+ggplot(mpg,aes(cty,class,color=class)) + 
+#Turn off the shape of outlying point
+geom_boxplot(color="black", outlier.shape=NA) + 
+geom_jitter(width=0,height=0.4) +
+scale_color_brewer(palette="Set2") +
+#I think the x-axis should be labeled as "City mpg" and y-axis should be labeled as "Car Class"

```

Comment:
That's a good call, I didn't noticed that. I instead flipped my coordinate and set height = 0. But I think yours makes more sense! Also it seems height = 0.4 is the default setting, so you may get same results without setting it?